napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
```

Comment:
I moved this one to the `Napi` namespace and into the public header. (And also renamed the namespace to `details`.)